src/utils/verifiable-credential/types.ts

```diff
@@ -0,0 +1,66 @@
+import type { Hex } from '@metamask/utils';
+
+export enum TrustworthinessScope {
+  Honesty = 'Honesty',
+  SoftwareSecurity = 'Software security',
+  SoftwareDevelopment = 'Software development',
+}
+
+export enum SnapCurrentStatus {
+  Endorsed = 'Endorsed',
+  Disputed = 'Disputed',
+}
+
+export enum SnapStatusReasonType {
+  Endorse = 'Endorse',
+  Dispute = 'Dispute',
+}
+
+export enum TrustCredentialType {
+  TrustCredential = 'TrustCredential',
+  VerifiableCredential = 'VerifiableCredential',
+  StatusCredential = 'StatusCredential',
+}
+
+export type PKHDid = `did:pkh:${Hex}`;
+
+export type SnapDid = `snap://${string}`;
+
+export type Did = PKHDid | SnapDid;
+
+export type StatusReason = {
+  type: SnapStatusReasonType;
+  value: string[];
+};
+
+export type Trustworthiness = {
+  scope: TrustworthinessScope;
+  level: number;
```

Comment:
regarding the `level` here, is level only number 2, 1, 0, -1, 2, or we can put any number here? i remember Dayan mention only 5 value here.

Reply:
i think we can leave it to validation part later